OWNERS_ALIASES

```diff
@@ -1,4 +1,4 @@
-# See the OWNERS docs: https://git.k8s.io/community/docs/devel/owners.md
+# See the OWNERS docs: https://git.k8s.io/communitydocs/devel/owners.md
```

Comment:
Seeing this part of the diff made me check both versions of the links and it looks like neither currently work. It looks like it should be https://git.k8s.io/community/contributors/guide/owners.md instead

Reply:
Fixing :)